app/controllers/v0/ppiu_controller.rb

```diff
@@ -17,6 +17,7 @@ def index
 
     def update
       response = service.update_payment_information(pay_info)
+      Rails.logger.info('[PPIUUpdate]:requestcompleted', sso_logging_info)
```

Comment:
I think these should follow a format that looks more like this:
'PPIUController#update request completed'